duckduckgo/iOS
# Add tos to site score

Reviewer: Chris
Asana: https://app.asana.com/0/414235014887631/426432715344337

**Description**:
Adds TOS to the privacy grade

**Steps to test this PR**:
1. Visit amazon.com and notice the site score logs now contain an entry `tos: d_1` meaning the amazon has a terms score of d and this has added a point to the site score
2. Visit disqus.com and notice the site score logs now contain an entry `tos: a_-1`

NOTE: I've followed the approach used by extensions but this performs matching by dropping the url's tld, which isn't perfect. I've opened a task to discuss and update this separately https://app.asana.com/0/414235014887631/430675284089081.

###### Reviewer Checklist:


###### PR DRI Checklist:
